kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/WebClientConfigurator.java

```diff
@@ -31,6 +31,10 @@ public class WebClientConfigurator {
 
   public WebClientConfigurator() {
     configureObjectMapper(defaultOM());
+    var httpClient = HttpClient
+        .create()
+        .proxyWithSystemProperties();
+    builder.clientConnector(new ReactorClientHttpConnector(httpClient));
```

Comment:
This would create an httpclient twice if `configureSsl` method is called (see line 98)

Reply:
HttpClient creation should be cheap. Its instance is duplicated during configuration:

<img width="837" alt="Screenshot 2023-07-18 at 09 39 14" src="https://github.com/provectus/kafka-ui/assets/3982146/17634ad3-6162-49ff-8973-2bc1fd775f0a">

Anyway, your recommendation made the code a bit more DRY.